component/media/css/bootstrap.css

```diff
@@ -90,7 +90,7 @@
 	bottom: -0.25em;
 }
 .jevbootstrap img {
-	width: auto\9;
```

Comment:
I removed it because it's not valid CSS? See:
https://jigsaw.w3.org/css-validator/validator

I assumed a typo, if we are adding I.e. Fixes we should be loading a special ie CSS file